src/source_sender/mod.rs

```diff
@@ -48,7 +48,25 @@ impl Builder {
         }
     }
 
-    pub fn add_output(&mut self, output: Output) -> LimitedReceiver<EventArray> {
+    pub fn add_output(&mut self, output: TransformOutput) -> LimitedReceiver<EventArray> {
```

Comment:
```suggestion
    pub fn add_transform_output(&mut self, output: TransformOutput) -> LimitedReceiver<EventArray> {
```

To better reflect the mirroring of the functions?

Reply:
Turns out it's not used at all.